pkgs/applications/terminal-emulators/kitty/themes.nix

```
@@ -5,13 +5,13 @@
 
 stdenvNoCC.mkDerivation {
   pname = "kitty-themes";
-  version = "unstable-2024-04-23";
+  version = "unstable-2024-05-28";
```

```suggestion
  version = "0-unstable-2024-05-28";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming